ci/benchmarks/partial-conv/esm2_pretrain.yaml

```
@@ -0,0 +1,42 @@
+time_limit: 14400
+script_args:
+  # All arguments referenced in the script string must be specified here. Please reference the script arguments by ${kay_name} or {key_name}
+  # Arguments not referenced in the script string must have the 'arg' field specified.
+  model:
+    value: esm2
+  variant:
+    value: pretrain
+  precision:
+    value: bf16-mixed
+  nodes:
+    value: 4
+  gpus:
+    value: 8
+  batch_size:
+    value: 16
+  max_steps:
+    value: 10000
+script: |-
+    export NVTE_FUSED_ATTN=1; export NVTE_FLASH_ATTN=0; python scripts/${variant}/${model}/${model}_${variant}.py \
```

```suggestion
    python scripts/${variant}/${model}/${model}_${variant}.py \
```
@jstjohn was seeing issues with these ENV variables (at least with our version of CUDNN), so we've removed them for now

---

@dorotat-nv yes, also note that the script paths are pretty different now. I believe there's a CLI arg thats like `train_esm2` https://github.com/NVIDIA/bionemo-framework/blob/main/sub-packages/bionemo-esm2/pyproject.toml#L24 

We also have the new `cli` which in two steps creates a config and runs the config. Perhaps it would be good to save our JET config settings in a `.json` file and instead just run that: https://github.com/NVIDIA/bionemo-framework/blob/main/sub-packages/bionemo-esm2/pyproject.toml#L21 cc @skothenhill-nv

---

what's the difference between `bionemo-esm2-train` and `train_esm2`?

---

`bionemo-esm2-train` goes through the pydantic interface and shares a `train` method with Geneformer. We can do the same thing for inference.

In anycase I'd definitely prefer if we prioritized using the pydantic interface for things.

---

@skothenhill-nv, given that the Pydantic interface has been under development recently, I've decided not to change the training command for now. I will wait until it stabilizes.